canonical-docs/conjure-up-docs
# Add python app for serving docs from this repo

Add app to serve docs from this repo directly.

## QA

Start the server with `./run`
The site should be available at http://localhost:8204
Verify the content shows correctly, including images. (There may be small styling tweaks from an updated theme)
